_caseStudies/2020-05-10-united-nations-lgbtqi-staff-platform.md

```diff
@@ -114,7 +114,7 @@ The open source software Pybossa can analyze voice clips or transcription text.
 
 ### Sensemaker
 
-While not open source, the company Cognitive Edge has a tool called [Sensemaker](https://sensemaker.cognitive-edge.com/product-category/sensemakerapplications/) that uses qualitative data collection methods that might be relevant. Sensemaker collects stories in snippets, and allows people to analyze their own contributions using a predefined set of tools. This allows the company to quickly derive signals based on cognitive frameworks.
+While not open source, the company Cognitive Edge has a tool called [Sensemaker](https://thecynefin.co/sensemaker/) that uses qualitative data collection methods that might be relevant. Sensemaker collects stories in snippets, and allows people to analyze their own contributions using a predefined set of tools. This allows the company to quickly derive signals based on cognitive frameworks.
```

Comment:
Do we want to reflect the change in the company name as well? https://thecynefin.co/about-us/ If yes, I suggest rephrasing to: 

While not open source, the Cynefin Company (formerly known as Cognitive Edge) has a tool called [Sensemaker](https://thecynefin.co/sensemaker/) that uses qualitative data collection methods that might be relevant. Sensemaker collects stories in snippets, and allows people to analyze their own contributions using a predefined set of tools. This allows the company to quickly derive signals based on cognitive frameworks.